save-common/src/commonMain/kotlin/org/cqfn/save/test/TestDtoForBatch.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqfn.save.test
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property filePath
+ * @property testSuiteId
+ * @property id
+ */
+@Serializable
+data class TestDtoForBatch(
```

Comment:
Why do we need two different dtos for tests? Are we using an id somewhere when dealing with batches?